deploy/cr.yaml

```diff
@@ -226,7 +226,7 @@ spec:
   proxy:
     pgBouncer:
       replicas: 3
-      image: perconalab/percona-postgresql-operator:main-ppg16-pgbouncer
+      image: percona/percona-postgresql-operator:2.5.0-ppg16-pgbouncer
```

Comment:
Please use tags with product versions like:
`perconalab/percona-postgresql-operator:2.4.1-ppg16.3-pgbouncer1.22.1`
you can find examples in https://github.com/percona/percona-postgresql-operator/blob/v2.4.1/deploy/cr.yaml